src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests/EventHandlers/CohortTransferApprovalRequestedEventHandlerTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using MediatR;
+using Moq;
+using NServiceBus;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Application.Commands.AddTransferRequest;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetCohortSummary;
+using SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.Encoding;
+using SFA.DAS.NServiceBus.Services;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests.EventHandlers
+{
+    [TestFixture]
+    public class CohortTransferApprovalRequestedEventHandlerTests
+    {
+        public CohortTransferApprovalRequestedEventHandlerTestsFixture _fixture;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _fixture = new CohortTransferApprovalRequestedEventHandlerTestsFixture();
+        }
+
+        [TestCase(Party.Employer)]
+        [TestCase(Party.Provider)]
+        public async Task When_HandlingCommand_TransferRequestIsCreated(Party lastApprovingParty)
+        {
+            await _fixture.WithLastApprovalParty(lastApprovingParty).Handle();
+            _fixture.VerifyTransferRequestCreated();
+        }
+
+        [Test]
+        public async Task When_HandlingCommand_IfLastApprovalWasProvider_EmailIsSentToEmployer()
+        {
+            await _fixture.WithLastApprovalParty(Party.Provider).Handle();
+            _fixture.VerifyEmailSentToEmployer();
+        }
+
```

Comment:
Is it worthwhile ensuring no email is sent when the LastApprovingParty is Employer?